specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-privatepreview/transactions.json

```diff
@@ -0,0 +1,209 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-11-01-privatepreview",
```

Comment:
Could you change the version to `2020-11-01-preview` since it will be public? Also keep the folder name aligned to new api version.

Reply:
@asarkar84 please answer @raych1's question.

Reply:
@raych1 We are releasing the privatepreview version and would like to keep it as is.

Reply:
Talked with @asarkar84 offline, this version of swagger doesn't need to merge to public since it's private preview and private repository already has this change reviewed. I put `DONOTMERGE` label and let me know if there's any questions.